### PR TITLE
 filter_kubernetes: add per stream and per container exclude annotation support

### PR DESCRIPTION
**Pull request by yann-soubeyrand**
Based on PR https://github.com/fluent/fluent-bit/pull/1257.

### CODE REVIEW

**Comment by edsiper** [comment]
@yann-soubeyrand would you please rebase the changes on top of master?

**Comment by yann-soubeyrand** [comment]
> @yann-soubeyrand would you please rebase the changes on top of master?

@edsiper This PR depends on #1257 which I've just tried to rebase on master but had a problem with test results: why are / escaped?

**Comment by edsiper** [comment]
in JSON / are optionally escaped for HTML purposes-use case

**Comment by yann-soubeyrand** [comment]
@edsiper Then the test results should have been updated I guess ;-) Anyway, I've fixed it and I've rebased #1257 on master and this PR on #1257. Could you have a look at it please? And could you also have a look at my other PR please?

File: plugins/filter_kubernetes/kube_property.c [hunk 1]
```diff
@@ -125,56 +130,122 @@ static int prop_set_exclude(struct flb_kube *ctx, struct flb_kube_meta *meta,
...
-    //  fluentbit.io/parser_stderr[-container: X
-    if (prop_cmp(FLB_UNIFIED_PARSER_ANNOTATION, sizeof(FLB_UNIFIED_PARSER_ANNOTATION)-1, prop, prop_len)) {
-        char *stream = 0;
-        const char *container = 0;
-        size_t stream_len =0;
-        size_t container_len = 0;
-        if (prop_cmp(FLB_STDOUT_PARSER_ANNOTATION, sizeof(FLB_STDOUT_PARSER_ANNOTATION)-1, prop, prop_len)) {
-            stream = "stdout";
-            stream_len = sizeof("stdout");
+    // Property can take the following forms:
```
[the remark this answers is not in the document]
Comment (yann-soubeyrand) [reply plugins/filter_kubernetes/kube_property.c]: Done ;-)

**Comment by edsiper** [comment]
thanks!

**Pull request merged**